recipe/meta.yaml

```diff
@@ -22,22 +22,25 @@ requirements:
     - pip
   run:
     - python
-    - alembic >=1.7
-    - ipywidgets >=7.6
-    - jinja2 >=3
+    - jupyter_packaging
```

Comment:
Maybe the `jupyter-packaging` requirement should be added in `host` section, like here https://github.com/conda-forge/jupyterlab-feedstock/blob/91e973dc9e99afa7b696eb0e9b1bf8d2316747aa/recipe/meta.yaml#L23